src/api/network/asio/Imp.cpp

```diff
@@ -377,26 +355,22 @@ auto Asio::Imp::Receive(
         bufData.second,
         [this, connection{space(id)}, type, bufData, address{endpoint.str()}](
```

Comment:
This is a huge capture by value. Would it not be better to capture space(id) and endpoint.str() to construct connection and address inside lambda body? Note that I am not suggesting a capture of id, which would be a grave risk.

Reply:
Im not sure if this is what you wanted. If i get it right we can capture id by copy (safe), but 
address{endpoint.str()} - is just copying string, right? so we cannot make it better since we have to copy it